### DISCUSSION TITLE
Please add these Privacy Coins: XMR & WOW

### DISCUSSION THREAD

**Discussion by marvelloz** [horizontalsystems/unstoppable-wallet-android#5354] [Ideas]
Hello Unstoppable, 
With the recent crack down on Tornado Cash and governments killing services that assist in helping us maintain our human right to privacy, I really feel it important that privacy coins be added to Unstoppable Wallet. Your wallet is very appropriately named and being open source and listed on F-Droid is AWESOME! 

Can you please ADD support for these coins in Unstoppable Wallet:
**1) Monero (XMR)**   
Homepage - https://www.getmonero.org/
Talk to Devs, visit - https://web.getmonero.org/community/workgroups/

**2) Wownero (WOW) - a fork of Monero**   
Homepage - https://wownero.org  
Talk to Devs - https://forum.wownero.com/

Many thanks for supporting privacy coins.

**Comment by abdrasulov** [comment]
#3302
